dialoguekit/nlu/models/diet_classifier_rasa.py

```diff
@@ -3,7 +3,7 @@
 More information about the DIET classifier
 https://rasa.com/docs/rasa/reference/rasa/nlu/classifiers/diet_classifier/
 
-A short description of how we use Rasa as a component library can be seen inn
+A short description of how we use Rasa as a component library can be seen in
 the docs/rasa_component_library.md
```

Comment:
Nit: drop "the"
That file is no longer there, can you link https://iai-group.github.io/DialogueKit/nlu.html#rasa-as-a-component-library instead?

Reply:
Done.